app/views/pages/_header.html.erb

```diff
@@ -3,20 +3,30 @@
 
         <%= link_to 'Employee Recognition Platform', root_path, class:"navbar-brand d-flex w-50 me-auto" %>
             <ul class="nav navbar-nav ms-auto w-100 justify-content-end">
-                <% unless employee_signed_in? %>
+                <% if admin_user_signed_in? %>
```

Comment:
Is it really needed here? I would see here simply _Admin Panel_ button taking you to admin dashboard, and there authentication checks if you can get in

Reply:
good point :) fixing